package-lock.json

```diff
@@ -23786,6 +23786,7 @@
       "resolved": "https://registry.npmjs.org/ts-jest/-/ts-jest-29.2.6.tgz",
       "integrity": "sha512-yTNZVZqc8lSixm+QGVFcPe6+yj7+TWZwIesuOWvfcn4B9bz5x4NDzVCQQjOs7Hfouu36aEqfEbo9Qpo+gq8dDg==",
       "dev": true,
+      "license": "MIT",
```

Comment:
Why is there a change here?

Reply:
That must have been auto applied when I npm installed

Reply:
There are thousands of them int the packge-lock

Reply:
I just see this one line. I'd rather not keep including unnecessary changes to our package-lock.json just to be most secure. I know this specific instance isn't a big deal, but let's just be strict.